src/physics/exact_solutions/exact_solution.cpp

```diff
@@ -0,0 +1,95 @@
+#include <deal.II/base/function.h>
+#include "exact_solution.h"
+
+namespace PHiLiP {
+
+// ========================================================
+// ZERO -- Returns zero everywhere; used a placeholder when no exact solution is defined.
+// ========================================================
+template <int dim, int nstate, typename real>
+ExactSolutionFunction_Zero<dim,nstate,real>
+::ExactSolutionFunction_Zero()
+        : ExactSolutionFunction<dim,nstate,real>()
+{
+    t = 0;//time_compare;
```

Comment:
Maybe use existing dealii::Function get_time, set_time, advance_time functions?

Reply:
I have changed `create_ExactSolutionFunction()` to take a time as input, which I think is sufficient for this PR's needs. I think those functions are probably useful to include in the future when working with the `dealii::Function` sections, though.